### DISCUSSION TITLE
Daily Export is out by a factor of 10

### DISCUSSION THREAD

**Discussion by LucidityCrash** [hultenvp/solis-sensor#261] [General]
Looks like the API / App / Web is returning a Daily export to grid that is 0.1 times reality again ... I seem to remember it doing this back when we were first writing this.

I've exported > 3 kw today as shown by looking at both the total export graph on the web page and seing how much it climbed, and the daily export register over modbus

Raised a ticket with solis support.

updating the workarounds.yaml to set correct_daily_on_grid_energy_enabled back to true will help.

**Comment by hultenvp** [comment]
Glad that at least the yaml seems to correct the issue again, however they seem to have an awful lot of regression recently.

Appreciated if you could drop a message here once it's solved again.

**Comment by LucidityCrash** [comment]
*sigh* 
Solis : it will be fixed by the end of next week ...
Solis (3 days later) : fixed now
Me : No its not
Solis (2 days later - this morning) : we fixed it day before yesterday, is it on web or api ?
Me  (looking at web graphs as I write) : Still broken and Both
Solis : please observe as requested and update.

**Comment by LucidityCrash** [comment]
dear ghads ... now it is because I've got charge from grid enabled ... and they turned it off remotely !

**Reply by alienatedsec** [reply]
I have found that their support is very limited and misleading on many occasions.

**Comment by hultenvp** [comment]
I suspect they have limited to no DTAP staging with sufficient (regression) test coverage in place.
Me neither, but I have the excuse this is a hobby project 🤣 

Maybe, if we find sufficient volunteers we should start our own cloud service ;-)
